src/libraries/System.Private.CoreLib/src/System/Text/ASCIIUtility.cs

```diff
@@ -860,24 +852,9 @@ private static unsafe nuint GetIndexOfFirstNonAsciiChar_Sse2(char* pBuffer, nuin
                     firstVector = Sse2.LoadAlignedVector128((ushort*)pBuffer);
                     secondVector = Sse2.LoadAlignedVector128((ushort*)pBuffer + SizeOfVector128InChars);
                     Vector128<ushort> combinedVector = Sse2.Or(firstVector, secondVector);
-
-                    if (Sse41.IsSupported)
-                    {
-                        // If a non-ASCII bit is set in any WORD of the combined vector, we have seen non-ASCII data.
-                        // Jump to the non-ASCII handler to figure out which particular vector contained non-ASCII data.
-                        if (!Sse41.TestZ(combinedVector, asciiMaskForTestZ))
-                        {
-                            goto FoundNonAsciiDataInFirstOrSecondVector;
-                        }
-                    }
-                    else
+                    if (VectorContainsNonAsciiChar(combinedVector))
```

Comment:
nit: Newline to separate data from the check makes it easier to read the code
```suggestion

                    if (VectorContainsNonAsciiChar(combinedVector))
```

Reply:
Rolled back changes to `GetIndexOfFirstNonAsciiChar_Sse2`